AirCasting/CreateSessionViews/TurnOnLocation/TurnOnLocationViewModel.swift

```diff
@@ -5,12 +5,24 @@ import Foundation
 
 class TurnOnLocationViewModel {
     
-    let locationHandler: LocationHandler
-    let bluetoothHandler: BluetoothHandler
-    let sessionContext: CreateSessionContext
-    let urlProvider: BaseURLProvider
+    private let locationHandler: LocationHandler
+    private let bluetoothHandler: BluetoothHandler
+    private let sessionContext: CreateSessionContext
+    private let urlProvider: BaseURLProvider
     
-    var mobileSessionContext: Bool {
+    var passURLProvider: BaseURLProvider {
```

Comment:
Maybe we should add a comment or a warning that this is only used to forward URLProvider and get rid of this asap